### PR TITLE
Support ResNet nchw/nhwc and refactor some codes

### PR DESCRIPTION
**Pull request by mingmingtasd**
PTAL, thanks! @fujunwei 
Now ResNet NHWC/NCHW can both pass on Linux/Windows as C++ or electron examples. All model tests can pass except on Windows DML backend, the related issue is #23.

### CODE REVIEW

File: node/src/ops/Pad.cpp [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+// limitations under the License.
+
+#include "ops/Pad.h"
+
+#include "Utils.h"
+
+namespace node { namespace op {
+
+    Napi::Value Pad::Build(const Napi::CallbackInfo& info, ml::GraphBuilder builder) {
+        // Operand pad(Operand input, optional PadOptions options = {});
```
Comment (fujunwei) [inline node/src/ops/Pad.cpp:22]: // Operand pad(Operand input, Operand pad, optional PadOptions options = {});

File: src/webnn_native/dml/GraphDML.cpp [hunk 2]
```diff
@@ -754,9 +754,9 @@ namespace webnn_native { namespace dml {
         ::dml::Expression padding = mExpression.at(inputsOperand[1].Get());
 
         // Workaround(mingming): If padding was added in mGraph, it must be used.
-        // Use time(NULL) to generate a unique name for the output node.
+        // Use (time(NULL) + mExpression.size()) to generate a unique name for the output node.
         // This may be a dml issue: https://github.com/microsoft/DirectML/issues/133.
-        std::string name = std::to_string(time(NULL));
+        std::string name = std::to_string(time(NULL) + mExpression.size());
```
Comment (fujunwei) [inline src/webnn_native/dml/GraphDML.cpp:759]: web can't make sure `time(NULL)`  is unique, so how about generating the identify like openvino? 
`std::string name = “pad” + mExpression.size();`

Comment (mingmingtasd) [reply src/webnn_native/dml/GraphDML.cpp:759]: Good suggestion, thanks! @fujunwei

File: examples/ResNet/Main.cpp [hunk 3]
```diff
@@ -0,0 +1,139 @@
...
+        << "    -l \"<layout>\"           "
+        << "Optional. Specify the layout: \"nchw\" or \"nhwc\". The default value is \"nchw\"."
+        << std::endl;
+    std::cout
+        << "    -n \"<integer>\"          "
+        << "Optional. Number of iterations. The default value is 1, and should not be less than 1."
+        << std::endl;
+}
+
+int main(int argc, const char* argv[]) {
```
[the remark this answers is not in the document]
Comment (mingmingtasd) [reply examples/ResNet/Main.cpp:41]: Yes, good suggestion, I will try to refactor and simplify the codes in main.cpp in another PR. Thanks! @fujunwei

**Review by fujunwei — APPROVED** [review]
LGTM.

**Comment by fujunwei** [comment]
Let's merge it, thanks.

**Pull request merged**